packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/auth/ApiKey.kt

```diff
@@ -0,0 +1,14 @@
+package io.realm.kotlin.mongodb.auth
+
+import io.realm.kotlin.types.ObjectId
+/**
+ * Class representing an API key for a User. An API can be used to represent the
+ * user when logging instead of using email and password.
+ * Note that a keys value is only available when the key is created, after that it is not
+ * visible. So anyone creating an API key is responsible for storing it safely after that.
+ * @param ObjectId, an id uniquely identifying the key.
+ * @param value, the value of this key. Only returned when the key is created.
+ * @param name, the name of the key.
+ * @param enabled, whether the key is enabled or not.
+ */
```

Comment:
```suggestion
/**
 * Class representing an API key for a [User]. An API key can be used to represent a
 * user when logging in instead of using email and password.
 * Note that the value of a key will only be available immediately after the key is created, after which point it is not
 * visible anymore. This means that keys returned by [ApiKeyAuth.fetch] and [ApiKeyAuth.fetchAll] will have a `null` [value]. Anyone creating an API key is responsible for storing it safely after that.
 * @param id an [ObjectId] uniquely identifying the key.
 * @param value the value of this key, only returned when the key is created, `null` otherwise.
 * @param name the name of the key.
 * @param enabled whether the key is enabled.
 */
```
Reworded some things plus added kdoc links - observe you also need to add `import`s for those linked classes used in the docs!